mantidproject/mantid
# ISISIndirectInelastic System test failure

Fixes #14395

Possible fix for the system test (`OSIRISFuryAndFuryFitMulti`) failure in `ISISIndirectInelastic` by updating the `II.OSIRISFuryFitMult.nxs.md5` reference file.

With the change in the reference file, the system test now passes on my machine (Windows) 
# To Test
- Ensure that the system tests pass on **ALL** platforms when run on the build servers.
